web/src/cypress/integration/pagination.ts

```diff
@@ -39,7 +39,7 @@ function testPagination(screen: ScreenFormat) {
 
 function testPaginating(label: string, url: string, create: createManyFunc) {
   let names: Array<string> = []
-  let nameSubstr: string = ''
```

Comment:
Is there a reason we are removing the type definition for the string but not for the array of strings

Reply:
This was removed by the `eslint --fix` since it's being set when declared it infers the type.

Reply:
Yeah, I suppose I'm curious why it's inferring some types but not others. Are only primitives inferred?

Reply:
Yea it's only inferring the primitives.